perceval/converters/circuit_to_graph_converter.py

```diff
@@ -0,0 +1,188 @@
+# MIT License
+#
+# Copyright (c) 2022 Quandela
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# As a special exception, the copyright holders of exqalibur library give you
+# permission to combine exqalibur with code included in the standard release of
+# Perceval under the MIT license (or modified versions of such code). You may
+# copy and distribute such a combined system following the terms of the MIT
+# license for both exqalibur and Perceval. This exception for the usage of
+# exqalibur is limited to the python bindings used by Perceval.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# ========================================
+# Main processing
+# 1. Function Qiskit gate based circuit => list of gates and modes they act on
+# 2. Convert the previous list to a weighted graph
+# 3. Cut the graph the best way possible
+# 4. Plot the graph
+import numpy as np
+import matplotlib.pyplot as plt
+import networkx as nx
+from qiskit import QuantumCircuit
+from perceval.utils.qmath import kmeans
+
+
+def gates_and_qubits(qiskit_circuit):
+    '''''
+    :rtype gates_names: list of strings
+    :rtype gates_qubits: list of lists of ints
+    '''''
+    # will be used in both classes.
+    gates_names = []
+    gates_qubits = []
+    for instr in qiskit_circuit.data:
+        # Extract gate name and qubits
+        gates_names.append(instr[0].name)
+        qubits = instr[1]
+        qubit_numbers = [qiskit_circuit.find_bit(q).index for q in qubits]
+        gates_qubits.append(qubit_numbers)
+    return gates_names, gates_qubits
+
+
```

Comment:
Suggestion: include `CircuitToGraphConverter` class in the `__init__` of converters following other converter classes. It will allow you import by using `from perceval.converters import CircuitToGraphConverter`

Reply:
Is it normal that when I try to run the __init__ after including it, I get 
Traceback (most recent call last):
  File "C:\DEV\Perceval\perceval\converters\__init__.py", line 30, in <module>
    from .qiskit_converter import QiskitConverter
ImportError: attempted relative import with no known parent package
?